README.md

```diff
@@ -26,7 +28,8 @@ java -jar <path-to-datacommons-import-tool-jar-with-dependencies.jar>
     - For MacOS: ```brew install maven```
 2. The tools use protobuf and require that `protoc` be installed.
     - For MacOS: ```brew install protoc```
-3. Make sure Java 11 is installed (you can install it from [here](https://www.oracle.com/java/technologies/javase-downloads.html#javasejdk))
+3. Make sure Java 11+ (but not Java 16) is installed
```

Comment:
For development we can say 11 or 12.  Yes, for dev we can be more strict about versions.